roles/satellite-clone/defaults/main.yml

```diff
@@ -10,3 +10,4 @@ disable_firewall: false
 run_katello_reindex: false
 run_pre_install_check: true
 register_to_portal: true
+satellite_migration: false
```

Comment:
I'm in favor of calling this rhel_migration as that is more accurate to what it is doing and may be more intuitive for the user